src/app/static/src/app/components/Stepper.vue

```diff
@@ -19,7 +19,7 @@
         </div>
         <stepper-navigation v-bind="navigationProps"/>
         <hr/>
-        <warning-alert v-if="activeStep !== 3" :warnings="activeStepWarnings"></warning-alert>
+        <warning-alert v-if="activeStep !== 3" :warnings="activeStepWarnings" :active-step="activeStep"></warning-alert>
```

Comment:
It feels a bit inconsistent that we're passing active step warnings as a prop, but then also passing the active step itself to figure out which mutation to use. I think either the `WarningAlert` just receives the active step as a prop and figures out both which warnings to display and which clear handler to use, or both the clear handler and the warnings object get created here and passed as props. No strong feelings as to which way we resolve it.

Reply:
Agreed. My instinct would probably be to keep the child component simple and let the parent Stepper component know how to clear warnings for each step (so the child would emit a 'clearWarnings' event with a step parameter which the parent would handle).